django_scrubber/scrubbers.py

```diff
@@ -100,8 +100,12 @@ def __call__(self, field):
 class Faker(object):
     INITIALIZED_PROVIDERS = set()
 
-    def __init__(self, provider):
+    def __init__(self, provider, *args, **kwargs):
         self.provider = provider
+        self.provider_args = args
+        self.provider_kwargs = kwargs
```

Comment:
we probably don't need to store these if they're only used for generating the hash key?

Reply:
We need them, they're to be used in the FakeData generation (see next comment). Otherwise you can't pass the args/kwargs to the faker provider.

Reply:
ah yes, you're right.